packages/test-utilities/src/createTestContainer.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Creates a dummy container to be passed as an `attachTo` option into enzyme's `mount` function.
+ * Enables actual JSDOM introspection.
+ * Make sure you call `removeTestContainer` at the end of your test to clean up after yourself.
+ */
+export function createTestContainer() {
+  const testContainer = document.createElement('div');
+  document.body.appendChild(testContainer);
+
+  const removeTestContainer = () => {
+    if (testContainer && testContainer.parentNode) {
```

Comment:
q: is the 1st part of the check needed `if (testContainer ` ? testContainer was set already

Reply:
A: someone can remove this container within `test` by accident or on purpose, so we need to doublecheck